packages/side-nav/src/vaadin-side-nav-item.js

```diff
@@ -66,7 +67,11 @@ class SideNavItem extends ElementMixin(ThemableMixin(PolylitMixin(LitElement)))
   /** @protected */
   render() {
     return html`
-      <a href="${this.path}" part="item" aria-current="${this.active ? 'page' : false}">
+      <a
+        href="${ifDefined(this.path ? this.path : undefined)}"
```

Comment:
I updated the web component so that we now can support empty string if explicitly set.

```suggestion
        href="${ifDefined(this.path)}"
```